src/Routes/Repositories/FilterDropdown.js

```diff
@@ -30,20 +24,26 @@ const FilterDropdown = ({ filters, dropdown, setDropdown }) => {
         <Select
           variant="single"
           aria-label="Select input for filters"
-          width="250px"
+          width="180px"
```

Comment:
it's possible not be px ? i mean this can cause difficults on screens with different resolution.